templates/download/desktop/index.html

```diff
@@ -168,35 +172,44 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
           <h2>Ubuntu {{ releases.latest.full_version }}</h2>
         </div>
         <div class="p-image-wrapper u-hide--small u-hide--medium">
-          {{ image(url="https://assets.ubuntu.com/v1/91be1162-Oracular%20Oriole%20Mascot%20transparent%20background.png",
-          alt="Oracular Oriole",
-          width="200",
-          height="200",
-          hi_def=True,
-          loading="auto") | safe
+          {{ image(url="https://assets.ubuntu.com/v1/ce1666ec-pp-master-orange-hex.svg",
+                    alt="Plucky Puffin",
+                    width="150",
+                    height="150",
+                    hi_def=True,
+                    loading="lazy") | safe
           }}
         </div>
       </div>
       <div class="col">
         <div class="p-section--shallow">
           <p>
-            The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 24.10 comes with nine months of security and maintenance updates, until July 2025.
+            The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 25.04 comes with nine months of security and maintenance updates, until January 2026.
           </p>
           <hr class="p-rule--muted" />
           <p class="u-responsive-realign">
             <a class="p-button--positive"
                href="/download/desktop/thank-you?version={{ releases.latest.full_version }}&amp;architecture=amd64"
                onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });">Download
-              {{ releases.latest.full_version }}
+              {{ releases.latest.full_version }}&nbsp;(AMD64)
             </a>
             {% if releases.latest.iso_download_size %}
-            <span class="u-text--muted">{{ releases.latest.iso_download_size }}</span>
+              <span class="u-text--muted" style="margin-right: 2rem;">{{ releases.latest.iso_download_size }}</span>
+            {% endif %}
+            <a class="p-button"
+               href="https://cdimage.ubuntu.com/releases/{{ releases.latest.full_version }}/release/ubuntu-{{ releases.latest.full_version }}-desktop-arm64.iso"
+               onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });">Download
+              {{ releases.latest.full_version }}&nbsp;(ARM64)
+            </a>
+            {% if releases.latest.iso_download_size %}
+              <span class="u-text--muted">{{ releases.latest.arm64_iso_download_size }}</span>
             {% endif %}
             <script>
               performance.mark("Download (Desktop) button rendered")
             </script>
           </p>
           <p class="p-text--small">
+            <i>AMD64 for Intel or AMD 64-bit architectures, ARM64 for ARM 64-bit.</i><br />
```

Comment:
Not a fan of using line breaks like this. From [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/br): "Creating separate paragraphs of text using `<br>` is not only bad practice, it is problematic for people who navigate with the aid of screen reading technology. Screen readers may announce the presence of the element, but not any content contained within `<br>`s. This can be a confusing and frustrating experience for the person using the screen reader."